woocommerce/woocommerce
# Prevent search request in the Extensions > My Subscriptions page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Noticed we send an extra search request in the My Subscriptions tab and remove it.



### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch and get a new build
2. Open dev tools and load [the **My Subscriptions** page](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions)
3. Check the network tab for search requests and see that we only send three requests
    - It's normal that we send three search requests on page load, that's to preload data for browse, themes and business services tabs. 
5. If not connected, connect your store to WooCommerce.com and confirm you can see your subscriptions
6. Please test around this issue



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Prevent search request in the Extensions > My Subscriptions page

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
![image](https://github.com/user-attachments/assets/07155a9c-4aae-4c86-ac4d-c2a7af9f2377)
